tendrl/node_agent/objects/cluster_message/__init__.py

```diff
@@ -10,6 +10,9 @@ def __init__(self, **cluster_message):
         self.value = 'clusters/%s/Messages/%s'
         self._etcd_cls = _ClusterMessageEtcd
 
+    def load_definition(self):
```

Comment:
overloading load_definitions is a way to bypass the definitions system (i.e you dont have to define the object/flow/atom in the definitions file)